docs/guides/implementation-guides/how-to-implement-dependency-injection.md

```diff
@@ -7,9 +7,120 @@ import MvvmArchitectureDiagram from '/img/gitbook-import/assets/image (3) (1) (2
 
 # How To Implement Dependency Injection
 
-Content in preparation.
+:::info
+For views you can make a simple [ViewLocator](../../concepts/view-locator) which will automatically resolve the view from the Locator.
+:::
 
 This guide will show you how to use Dependency Injection (DI) with _Avalonia UI_ and the MVVM pattern. 
 
 <img src={MvvmArchitectureDiagram} alt=""/>
 
+Let's assume that your MainViewModel as a dependency on MyService. The contructor for ViewModel should looks like this:
+```csharp
+private readonly MyService _myService;
+
+public MapsViewModel(MyService myService)
```

Comment:
Is there a particular reason this is called `MapsViewModel`? It does not seem relevant to anything here. I would suggest using a more generic name if it's not needed.

Reply:
It was from my code, and I missed to change this, it's now corrected